README.md

```diff
@@ -46,7 +46,7 @@ main()
 
 ### gitRepoIsUpToDateReturnObject
 
-[lib/index.js:28-91](https://github.com/ibi-group/git-repo-is-up-to-date/blob/8581d574a0dfab701da75e9699d8576ef25f6f87/lib/index.js#L7-L18 "Source code on GitHub")
+[lib/index.js:28-119](https://github.com/ibi-group/git-repo-is-up-to-date/blob/a37f5016ea8291a0cd7a42998693d476bb407882/lib/index.js#L7-L18 "Source code on GitHub")
```

Comment:
For the object descriptor, I think the link text should indicate lines 7:18.

Reply:
This sectino is auto-generated by the commaned `yarn generate-docs`, however it is not quite right as you point out. I ended up rearranging some comments so that the line numbers match a little better in 0e1ef61b87da99a059e8f660e536eec3b83ad8db.